sktime/tests/_config.py

```diff
@@ -85,6 +85,7 @@
 from sktime.transformations.series.compose import OptionalPassthrough
 from sktime.transformations.series.outlier_detection import HampelFilter
 from sktime.transformations.series.boxcox import BoxCoxTransformer
+from sktime.transformations.series.multivariate_compositor import MultivariateCompositor
```

Comment:
Maybe it makes sense to have this in `sktime.transformations.series.compose`? Now there is a second path that contains `compositor` which is a bit rudimentary?

Reply:
Thanks for your feedback, @aiwalter!
I'll move it to the compose directory.